config/settings/sentry.py

```diff
@@ -0,0 +1,42 @@
+from config.env import env
+from config.django.base import SERVER_ENV
+
+SENTRY_DSN = env("SENTRY_DSN", default="")
+
+if SENTRY_DSN:
```

Comment:
local에서 sentry 보고는 생략되도록 해당 코드 적용해주시면 좋을 것 같습니다

```suggestion
if SENTRY_DSN and SERVER_ENV != "config.django.local":
```

Reply:
예외 핸들러 모듈에서

if SERVER_ENV in ["config.django.dev", "config.django.prod"]:
    sentry_sdk.capture_exception(exc)

처럼 설정을 해서, 중복되는 환경 변수 검사가 발생하여 해당 부분을 수정했습니다.